docker/chromium/ml-with-gpu/Dockerfile

```diff
@@ -23,8 +23,22 @@ WORKDIR /data
 
 # The dependencies installation commands below follow
 # https://www.tensorflow.org/install/gpu#ubuntu_1604_cuda_101.
-
-# Install development and runtime libraries (~4GB)
+#
+# Add NVIDIA package repositories.
+# Add HTTPS support for apt-key.
+RUN apt-get install -y gnupg-curl && \
+    wget https://developer.download.nvidia.com/compute/cuda/repos/ubuntu1604/x86_64/cuda-repo-ubuntu1604_10.1.243-1_amd64.deb && \
+    apt-key adv --fetch-keys https://developer.download.nvidia.com/compute/cuda/repos/ubuntu1604/x86_64/7fa2af80.pub && \
+    dpkg -i cuda-repo-ubuntu1604_10.1.243-1_amd64.deb && \
+    apt-get update && \
+    wget http://developer.download.nvidia.com/compute/machine-learning/repos/ubuntu1604/x86_64/nvidia-machine-learning-repo-ubuntu1604_1.0.0-1_amd64.deb && \
+    apt install -y ./nvidia-machine-learning-repo-ubuntu1604_1.0.0-1_amd64.deb && \
+    apt-get update
+
+# Delete the files we downloaded and used above.
+RUN rm -f cuda-repo-* nvidia-machine-learning-repo-*
```

Comment:
Put this in the same RUN above to actually prevent them from being included in the image. Otherwise a layer still gets created with those existing.

Reply:
good call, done!